src/sider/sykmeldinger/sykmelding/avvisteSykmeldinger/AvvistSykmeldingAlert.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from "react";
+import { SykmeldingOldFormat } from "@/data/sykmelding/types/SykmeldingOldFormat";
+import * as avvisningsregelnavn from "../../../../utils/sykmeldinger/avvisningsregelnavn";
+import { Alert, BodyShort, Box, Heading, List } from "@navikt/ds-react";
+
+const avvistSykmeldingFilter = (sykmelding: SykmeldingOldFormat) => {
+  return sykmelding.behandlingsutfall.ruleHits.filter((rule) => {
+    return rule.ruleStatus === null || rule.ruleStatus === "INVALID";
+  });
+};
+
+interface BegrunnelseTekstProps {
+  sykmelding: SykmeldingOldFormat;
+}
+
+const BegrunnelseTekst = (begrunnelseTekstProps: BegrunnelseTekstProps) => {
```

Comment:
Samme som over her 😊
```suggestion
function BegrunnelseTekst(begrunnelseTekstProps: Props) {
```